### DISCUSSION TITLE
Search by stars doesn't work

### DISCUSSION THREAD

**Discussion by Masynchin** [community/community#27507] [Repositories]
Search with sorted by stars doesn't work. For example, there are most starred Ruby repos according to your searching engine:

<img width="1440" alt="Repos with one thousand stars appeared at the top of the search" src="https://user-images.githubusercontent.com/47028153/181605215-3011c304-a4a8-496a-a8f4-f9a04a4b0426.png">

**Comment by radar-blip** [comment]
The categories are not changing. The repos in the right column are changing accordingly. Its working correctly for me.

**Reply by Masynchin** [reply]
> Its working correctly for me.

I am searching for repos on Ruby, and sort it by stars. How it can be correctly, if there is no Homebrew or any other top repos?

**Reply by radar-blip** [reply]
What I mean is: according to your search, the repos with the most stars is at the top of the list. 
If you're searching for homebrew, you could add that to your search.

**Reply by Masynchin** [reply]
> ... according to your search, the repos with the most stars is at the top of the list.

I am searching for repos written in Ruby sorted by stars. Does Homebrew written in Ruby - yes. Does it have enough stars to be in the first results - yes. Where is it?

> If you're searching for homebrew, you could add that to your search.

I don't have to specify Homebrew in my search query, it should be on the first page, if I am searching for most starred repos on Ruby.

**Comment by JeromeBeckett** [comment]
Still not fixed :/
